x-pack/solutions/security/plugins/security_solution/public/detections/links.ts

```diff
@@ -29,15 +29,15 @@ export const alertsLink: LinkItem = {
 };
 
 export const alertSummaryLink: LinkItem = {
-  capabilities: [[`${SECURITY_FEATURE_ID}.show`, `${SECURITY_FEATURE_ID}.alerts_summary`]],
+  id: SecurityPageName.alertSummary,
+  path: ALERT_SUMMARY_PATH,
+  title: ALERT_SUMMARY,
+  capabilities: [[`${SECURITY_FEATURE_ID}.show`, `${SECURITY_FEATURE_ID}.external_detections`]],
```

Comment:
Why changing the order here? The properties were alphabetically ordered and now they're not 😆

Reply:
Thanks @PhilippeOberti 👍 
I'll change the order, that must have been done by accident :) 

Regarding the access to pages, good observation, I think there's still some work to be done on our side. I just checked all of them and only `assets` is accessible as it would be in Complete/Essentials. However it's still in progress, the rest of the above are either redirecting to get_started, showing upselling, missing privileges, or just empty. 

Now trying to make sure rules/alerts are covered since we'll introduce 2 new pages instead.

cc: @ashokaditya - could you take care of the list above and make sure we redirect everywhere?

Reply:
Yup the other pages can be tackle as part of follow PRs for sure!